BookReaderDemo/index.html

```diff
@@ -25,6 +25,8 @@
       <li><a href="demo-plugin-menu-toggle.html">Plugin: Full screen menu toggle</a></li>
       <li><a href="immersion-mode.html">Start in immersion (fullscreen) mode</a></li>
       <li><a href="immersion-1up.html">Start in immersion mode on 1up default item</a></li>
+      <li><a href="demo-generic.html?ocaid=theworksofplato01platiala">demo-ia-plato with generic demo</a></li>
+      <li><a href="demo-generic.html?ocaid=adventuresofoli00dick">demo-ia-olivertwist with generic demo</a></li>
```

Comment:
Replace the previous files, `demo-ia-plato` and `demo-ia-olivertwist` with these links.